### PR TITLE
update rocksdb version

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
update rocksdb version

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
@BusyJay @siddontang PTAL

File: librocksdb_sys/build.sh [hunk 1]
```diff
@@ -105,14 +105,14 @@ function compile_rocksdb() {
         return
     fi
 
-    version=v4.9
-	vernum=4.9
+    version=4.12
```
Comment (BusyJay) [inline librocksdb_sys/build.sh]: Why not set as `4.12.fb` instead?

File: librocksdb_sys/build.sh [hunk 2]
```diff
@@ -105,14 +105,14 @@ function compile_rocksdb() {
...
+	vernum=4.12
     echo building rocksdb-$version
-    rm -rf rocksdb-$vernum
-    download https://github.com/facebook/rocksdb/archive/$version.tar.gz rocksdb-$version.tar.gz ba59206bd041a1d37157b9423d53572e
-    tar xf rocksdb-$version.tar.gz
+    rm -rf rocksdb-$vernum.fb
+    download https://github.com/facebook/rocksdb/archive/$version.fb.zip rocksdb-$version.fb.zip 83a0d8d1ef662659364427335a342333
```
Comment (BusyJay) [inline librocksdb_sys/build.sh]: Why use zip package?

**Comment by BusyJay** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**